src/components/Login/index.tsx

```diff
@@ -1,94 +1,181 @@
 import React from 'react';
-import { request } from 'http';
+import Banner1 from '../../assets/Banners/Calcumon-Banner1.png'
+
+import { withRouter, Redirect } from 'react-router-dom'
+
 import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
-import {Link} from 'react-router-dom'
+import { ReducersMapObject, Reducer, compose } from 'redux'
+
+import { logIn, LOGIN_STATE } from '../../actions/authentication' 
+
+import { Link } from 'react-router-dom';
+
+import './styles/login.css';
+
+import { LOG_IN } from '../../constants/authConstants'
 
-import { setAuthentication } from '../../actions/authentication'
 
 // TODO: What will be in our LoginInterface?
-interface IErrorLoginState {
-    showErrorMessage: Boolean;
-    username: string;
-    password: string;
-    error: boolean;
-}
+//State TYPES
 
+
+//TYPES FOR props
```

Comment:
Remove this comment. Seems unnecessary because its already named Props. Same for StateProps below.